keyboards/handwired/steamvan/keymaps/default/keymap.c

```diff
@@ -0,0 +1,91 @@
+/* Copyright 2019 John M Daly <jmdaly@gmail.com>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+// This file is based on the default keyman the the MiniVan.
+
+
+// Each layer gets a name for readability, which is then used in the keymap matrix below.
+// The underscores don't mean anything - you can have a layer called STUFF or any other name.
+// Layer names don't all need to be of the same length, obviously, and you can also skip them
+// entirely and just use numbers.
+
+#define _QW 0
```

Comment:
I'd recommend using an enum for your layer indexes, because it makes managing them easier.

Replace Lines 26-31 with:

```c
enum layer_names {
    _QW,
    _DV,
    _CM,
    _L1,
    _L2,
    _L3,
};
```

This is programmatically identical to the defines you currently have.

Reply:
Good call. Addressed in commit 3a639a04bd08b53e46317719fb8d54692cb45c51